frontend/src/Pages/ResourcesManagement.tsx

```diff
@@ -181,11 +282,12 @@ export default function ResourcesManagement() {
                             <h3 className="">Resources Preview</h3>
                             <div className="tooltip" data-tip="Add Collection">
                                 <button
-                                    className="btn btn-primary btn-circle btn-xs "
+                                    className="btn btn-primary btn-circle btn-xs"
                                     data-tip="Add Collection"
                                     onClick={() =>
                                         addCollectionModal.current?.showModal()
                                     }
+                                    disabled={!hasMadeModifications()}
```

Comment:
Is this necessary? I should be able to add new collections without having to make modifications.